README.md

```diff
@@ -1,50 +1,78 @@
-| **api**         | [Sample Azure Functions app](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
+| **api**         | [Fastify API](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
 | **react-app**   | [Sample React app](https://github.com/johnpapa/shopathome/blob/master/react-app)     | [react.shopathome.dev](https://react.shopathome.dev)     |
 | **svelte-app**  | [Sample Svelte app](https://github.com/johnpapa/shopathome/blob/master/svelte-app)   | [svelte.shopathome.dev](https://svelte.shopathome.dev)   |
 | **vue-app**     | [Sample Vue app](https://github.com/johnpapa/shopathome/blob/master/vue-app)         | [vue.shopathome.dev](https://vue.shopathome.dev)         |
 
-## Prerequisites
+## Run Shop at Home on Azure 
+> [!NOTE]
+> Ensure you fork the project to your GitHub account before you start deploying.
 
-- A GitHub account
-- [Node.js and Git](https://nodejs.org/)
-- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=academic-0000-jopapa) installed
-- The [Static Web Apps extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps&WT.mc_id=shopathome-github-jopapa&WT.mc_id=academic-0000-jopapa) installed
-- The [SWA CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=academic-0000-jopapa) installed
+[![Deploy to Azure button](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#view/Microsoft_Azure_PaasServerless/StarshotTemplateGallery.ReactView)
+
+1. Once you are signed in on Azure, under **Create new App Space**, select the **Deploy code from your GitHub repository** option
+1. Fill in the required information:
+
+    - *GitHub account* – Connect to your GitHub Account.
+    - *Organization* – Choose your organization.
+    - *Repository* – Select the shopathome repository from the dropdown.
+    - *Branch* – Choose a branch.
+    - *App Location* – Location of the application code in your repository. Choose a location depending on your framework of choice (./react-app, ./svelte-app, ./vue-app, ./angular-app).
+    - *App Space name* – Give your App Space resource a name.
+    - *Subscription* – Choose your subscription.
+    - *Region* – Choose an Azure region.
+  
+1. Click **Deploy**
+
+Once your App Space is provisioned successfully, open it to see your frontend component, and click on the link **Open app in browser** to view the frontend app on Azure Static Web Apps.
+
+<!--- See [the full tutorial](https://review.learn.microsoft.com/en-us/training/modules/introduction-code-to-cloud/4-exercise-deploy-fullstack-app-from-github?branch=pr-en-us-45512) to deploy your frontend. -->
+
+To add your backend:

```

Comment:
You won't need to do this as we would add the shop at home templates as solution templates. So, from create flow selecting the template would get you all components setup

Reply:
Got it! Thanks for the clarification